libgraph/include/katana/PropertyGraph.h

```diff
@@ -662,7 +672,13 @@ class KATANA_EXPORT PropertyGraph {
     return MakeResult(std::move(array));
   }
 
-  const GraphTopology& topology() const noexcept { return *topology_; }
+  void DropAllTopologies() noexcept {
```

Comment:
@witchel This is probably what you were asking about in Monday's meeting.

@mindv0rtex Please add a TODO to provide more fine-grained control of dropping specific topologies.

Reply:
My plan for the memory management RFC is that we will have many Managers, like a view manager, that communicate with the overall memory manager.  The overall memory manager might say to the view manager, please reduce your memory use, so it would be helpful to know that there are topologies that are not in use and safe to drop.  It would also be useful to know (approximately) how much memory that frees up. (All of this not for this PR, obviously).